### PR TITLE
GSoC: Adding Week 12 Blogpost

### PR DESCRIPTION
**Pull request by ganimtron-10**
image

### CODE REVIEW

**Comment by JoaoDell** [comment]
Hey @ganimtron-10, just reviewed your blogpost, which built fine, and everything seems ok. Thanks for describing your progress related to #832, I got a better understanding of what is goind on now :+1

**Review by JoaoDell — APPROVED** [review]
Hey @ganimtron-10, just reviewed your blogpost, which built fine, and everything seems ok. Thanks for describing your progress related to https://github.com/fury-gl/fury/pull/832, I got a better understanding of what is goind on now 👍

File: docs/source/posts/2023/2023-08-19-week-12-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+.. image:: https://github.com/fury-gl/fury/assets/64432063/b001f9d3-a5e8-45ad-8605-85df595b5654
+   :align: center
+   :alt: Text Overflow Before
+
+.. image:: https://github.com/fury-gl/fury/assets/64432063/d3c9c3a3-e601-45ab-8975-2b1e98acf1d3
+   :align: center
+   :alt: Text Overflow After
+
+**Addressing ZeroDivisionError:** The ``ZeroDivisionError`` emerged when the total number of values were same as the number of slots. The issue lay in the separation of these values for calculating the scrollbar's height parameter. Unfortunately, this calculation error occurred when this would return us zero while updating the scrollbar. To counter this, I implemented a conditional check to ascertain whether the value is zero or not.
```
Comment (tvcastillod) [inline docs/source/posts/2023/2023-08-19-week-12-praneeth.rst:33]: the total number of values _**was the same**_ as the number

**Review by tvcastillod — COMMENTED** [review]
Hi @ganimtron-10, the blogpost is clear and well-explained, now I understand better what PR #832 is about. I just left one comment for you to check.

**Comment by ganimtron-10** [comment]
Thanks @JoaoDell @tvcastillod for the review!

**Review by skoudoro — APPROVED** [review]
All good, Thanks @ganimtron-10, merging

**Pull request merged**